docs/guide/export.rst

```diff
@@ -31,8 +31,60 @@ to do inference in another framework.
 Export to ONNX
 -----------------
 
-TODO: help is welcomed!
+As of June 2021, ONNX format  `doesn't support <https://github.com/onnx/onnx/issues/3033>`_ exporting models that use the ``broadcast_tensors`` functionality of pytorch. So in order to export the trained stable-baseline3 models in the ONNX format, we need to first remove the layers that use broadcasting. This can be done by creating a class that removes the unsupported layers.
+
+For PPO, assuming a shared feature extactor is used:
+
+.. code-block:: python
+
+  from stable_baselines3 import PPO
+  import torch
+
+  class OnnxablePolicy(torch.nn.Module):
+    def __init__(self, extractor, action_net, value_net):
+        super(OnnxablePolicy, self).__init__()
+        self.extractor = extractor
+        self.action_net = action_net
+        self.value_net = value_net
+
+    def forward(self, input):
+        action_hidden, value_hidden = self.extractor(input)
+        return (self.action_net(action_hidden), self.value_net(value_hidden))
+
+  model = PPO.load("PathToTrainedModel\MyModel.zip")
```

Comment:
maybe just `PathToTrainModel.zip` to avoid being platform specific ? (windows here I presume)